cumulusci/tasks/salesforce/tests/test_activate_flow.py

```diff
@@ -0,0 +1,88 @@
+    @responses.activate
+    def test_half_activate_flow_processes(self):
+        cc_task = create_task(
+            ActivateFlow,
+            {
+                "developer_names": [
+                    "Auto_Populate_Date_And_Name_On_Program_Engagement",
+                    "ape",
+                ],
+            },
+        )
+        record_id = "3001F0000009GFwQAM"
+        activate_url = "{}/services/data/v43.0/tooling/sobjects/FlowDefinition/{}".format(
+            cc_task.org_config.instance_url, record_id
+        )
+        responses.add(
+            method="GET",
+            url="https://test.salesforce.com/services/data/v43.0/tooling/query/?q=SELECT+Id%2C+ActiveVersion.VersionNumber%2C+LatestVersion.VersionNumber%2C+DeveloperName+FROM+FlowDefinition+WHERE+DeveloperName+IN+%28%27Auto_Populate_Date_And_Name_On_Program_Engagement%27%2C%27ape%27%29",
+            body=json.dumps(
+                {
+                    "records": [
+                        {
+                            "Id": record_id,
+                            "DeveloperName": "Auto_Populate_Date_And_Name_On_Program_Engagement",
+                            "LatestVersion": {"VersionNumber": 1},
+                        }
+                    ]
+                }
+            ),
+            status=200,
+        )
+        data = {"Metadata": {"activeVersionNumber": 1}}
+        responses.add(method=responses.PATCH, url=activate_url, status=204, json=data)
+
+        cc_task()
+        self.assertEqual(2, len(responses.calls))
+
+    @responses.activate
+    def test_both_activate_flow_processes(self):
+        cc_task = create_task(
+            ActivateFlow,
+            {
+                "description": "Activates Flows identified by a given list of Developer Names",
+                "developer_names": [
+                    "Auto_Populate_Date_And_Name_On_Program_Engagement",
+                    "ape",
+                ],
+                "required": True,
+            },
+        )
+        record_id = "3001F0000009GFwQAM"
+        activate_url = "{}/services/data/v43.0/tooling/sobjects/FlowDefinition/{}".format(
+            cc_task.org_config.instance_url, record_id
+        )
+        responses.add(
+            method="GET",
+            url="https://test.salesforce.com/services/data/v43.0/tooling/query/?q=SELECT+Id%2C+ActiveVersion.VersionNumber%2C+LatestVersion.VersionNumber%2C+DeveloperName+FROM+FlowDefinition+WHERE+DeveloperName+IN+%28%27Auto_Populate_Date_And_Name_On_Program_Engagement%27%2C%27ape%27%29",
+            body=json.dumps(
+                {
+                    "records": [
+                        {
+                            "Id": record_id,
+                            "DeveloperName": "Auto_Populate_Date_And_Name_On_Program_Engagement",
+                            "LatestVersion": {"VersionNumber": 1},
+                        },
+                        {
+                            "Id": record_id,
```

Comment:
A real query result could not include two entries with the same record Id. This test works, but the setup differs from reality in a way that _might_ have impact (although I'm fairly sure it does not here).

I'd suggest adding a second record Id and URL. That will allow you to verify that exactly one activate call is made per record returned.